drop-analytics/Cargo.toml

```diff
@@ -8,6 +8,7 @@ slog = { workspace = true }
 serde = { workspace = true }
 serde_json = { workspace = true }
 anyhow = { workspace = true }
+uuid = { workspace = true }
```

Comment:
It would be nice to include this dependency only when the `moose_file` feature is on

Reply:
too many comments, gonna tackle those of higher priority